demisto_sdk/commands/lint/tests/test_pylint_plugin/xsoar_checker_test.py

```diff
@@ -0,0 +1,239 @@
+import astroid
+import pylint.testutils
+from demisto_sdk.commands.lint.resources.pylint_plugins import base_checker
+
+
+class TestPrintChecker(pylint.testutils.CheckerTestCase):
```

Comment:
why do we need a separate class for each test case?

Reply:
No real need, Thought to separate for order, when we will have more functionality in each plugin it might be better to have classes for each functionality.